pyproject.toml

```diff
@@ -50,7 +50,7 @@ xlrd = "*"
 openpyxl = "*"
 fsspec = "*"
 pydra-nipype1 = "^0.2"
```

Comment:
```suggestion
pydra-nipype1 = "^0.3"
```

Reply:
Thanks for looking into it 🙏 

This particular issue is gone now 🎉